cmd/tools/web-ui/server.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
```

Comment:
You should change a bit the server:
I have written a working example here: https://gist.github.com/Gsantomaggio/2b48f3e92298b1f42affff2cba97a5de
Let me know if you have questions